examples/dt/neutron/3cont_2comp-ipv6-all-geneve-dvr-stf-rally

```diff
@@ -0,0 +1,62 @@
+# Deployed Topology 3cont_2comp-ipv6-all-geneve-dvr-stf-rally
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+|           | ovn-router-flavors           |                    |
```

Comment:
what's the reason why router-flavors are enabled in this job but not in others? is there some test scenario that fails with it enabled?

Reply:
This is aligned with our existing d/s jobs. IIUC, we can have ovn-router-flavors enabled by default in all jobs, but usually we use neutron defaults and enable specific features where needed

Reply:
indeed, this may be a better question for the product - if we should have it enabled by default.

Reply:
IIRC this feature had enabling and disabling added and defaults to disabled since it had potential wide impact in case there is a bug, we didn't want it to affect many when only a few need these extra customized flavors.
Other jobs check no regressions exist when feature is disabled (most users), while this specific one tests when enabled.

Reply:
Marking this as Resolved, because this is mapped to unified job uni-delta-ipv6